test/experiment_test.rb

```diff
@@ -422,4 +422,29 @@ def test_no_storage
 
     assert_kind_of Verdict::Storage::MockStorage, e.storage
   end
+
+  def test_cleanup
+    redis = ::Redis.new(host: REDIS_HOST, port: REDIS_PORT)
+    storage = Verdict::Storage::RedisStorage.new(redis)
+    experiment = Verdict::Experiment.new(:cleanup_test) do
+      groups { group :all, 100 }
+      storage storage, store_unqualified: true
+    end
+
+    experiment.assign("something")
+    assert_operator redis, :exists, "experiments/cleanup_test"
+
+    experiment.cleanup
+    refute_operator redis, :exists, "experiments/cleanup_test"
```

Comment:
I wonder if there should be some `ensure` part at the end of this test to manually wipe the data from Redis in case something goes wrong in the test to ensure it doesn't leak between tests.